doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -50,11 +48,15 @@ Building SciPy
 
 #. (Optional) Enter ``conda list`` again. Note that the new virtual environment has no packages installed. If you were to open a Python interpreter now, you wouldn't be able to import ``numpy``, ``scipy``, etc...
 
-#. Enter ``conda install cython numpy pytest spyder pybind11``. |br| Note that we're only installing SciPy's build dependencies (and Spyder so we can use the IDE), but not SciPy itself.
+#. Enter ``conda install python=3.8 numpy pybind11 cython pytest gfortran_linux-64 gxx_linux-64 sphinx git``. |br| Note that we're installing SciPy's build dependencies and some other software, but not SciPy itself.
```

Comment:
@rgommers Does `mypy` help? Is there a reason to install `sphinx` without `matplotlib`?

Reply:
This renders as:
![image](https://user-images.githubusercontent.com/6570539/89811058-eb217280-daf2-11ea-8e35-59b8c9c511e0.png)

Is the inconsistent styling avoidable?

Reply:
> Does `mypy` help? Is there a reason to install `sphinx` without `matplotlib`?

The aim is to get a complete development environment here. So yes I'd like to add `mypy`. And good point about `matplotlib`, let's add that for doc building.

> Yes, this can be combined with creating the environment, and that is what an experienced user should do. But someone new to `conda` would benefit from understanding what these commands do separately.

Actually it is better practice to install everything at once. You're not guaranteed to get the same optimal solution if you chop it up.

Reply:
Re doc styling: you need to unindent the bullet list. The vertical bar on the left indicates that you added unwanted indentation, and Sphinx tries to render as a quote.

Reply:
> Re doc styling: you need to unindent the bullet list

I didn't see the extra space; it needed three instead of four.

Reply:
> So yes I'd like to add `mypy`.

OK. I've never used it, so I don't know what to write about it.